src/components/App/App.js

```diff
@@ -15,21 +15,22 @@ function mapStateToProps(state) {
   return {
     textColor: state.textColor,
     fontSize: state.fontSize,
+    fontUnitOfMeasure: state.fontUnitOfMeasure,
     isFontBold: state.isFontBold,
     backgroundColor: state.backgroundColor,
     accessibilityLevel: state.accessibilityLevel
   };
 }
 
 function App(props) {
-  const { textColor, fontSize, isFontBold,
+  const { textColor, fontSize, fontUnitOfMeasure, isFontBold,
           backgroundColor, accessibilityLevel } = props;
   const isUserInputValid = textColor.isValueValid &&
                            backgroundColor.isValueValid &&
                            fontSize.isValid;
   const fontSizeValue = parseInt(fontSize.value, 10);
   const accessibleContrastRatio = isUserInputValid ?
-    accessibleContrast(accessibilityLevel, fontSizeValue, isFontBold) : null;
+    accessibleContrast(accessibilityLevel, fontSizeValue, isFontBold, fontUnitOfMeasure) : null;
```

Comment:
Can we change the order of the last 3 args to: `fontSizeValue, fontUnitOfMeasure, isFontBold`? This is the order in the UI, so would be nice to keep it consistent.